package.json

```
@@ -71,6 +71,6 @@
   "scripts": {
     "test": "bower install && node ./node_modules/karma/bin/karma start --single-run --browsers PhantomJS"
   },
-  "version": "1.3.0",
+  "version": "1.2.3",
```

Can you update to the next minor and regenerate build files, please?
```suggestion
  "version": "1.3.1",
```